CONTRIBUTING.md

```diff
@@ -82,31 +146,41 @@ Before you submit your Pull Request (PR) consider the following guidelines:
 
 3. Follow our [Coding Rules](#rules).
 
-4. Run the entire test suite to ensure tests are passing.
+4. <a name="test-suite"></a>Run the entire test suite to ensure tests are passing.
 
     ```shell
-    cargo test
+    cargo test --all-features
     ```
 
-5. Commit your changes using a descriptive commit message that follows our [commit message conventions](#commit). Adherence to these conventions is necessary because release notes are automatically generated from these messages.
+5. <a name="code-quality-checks"></a>Run the following code quality checks locally so that the code is not only correct but also clean.
+
+    ```shell
+    cargo fmt --all -- --config imports_granularity=Crate,group_imports=One --check
+    cargo clippy --all-targets --all-features -- -D warnings
+    ```
```

Comment:
This does not cover all of our CI checks. This should be automated with a script proposed in #208